app/src/test/java/protect/card_locker/DatabaseTest.java

```diff
@@ -375,9 +375,9 @@ public void duplicateGroupName() {
         assertTrue(result);
         assertEquals(1, DBHelper.getGroupCount(mDatabase));
 
-        Group group = DBHelper.getGroup(mDatabase, "group one");
+        Group group = DBHelper.getGroupByName(mDatabase, "group one");
```

Comment:
We should also test if we can get a group by ID, as that's the actual canonical thing.